### PR TITLE
Rare slam sound

### PR DESCRIPTION
**Pull request by Waseemq1235**
# About the pull request

I come forward with a new wall-slam that has an extremely low chance of triggering due to the fact that it's literally the metal pipe falling sound.

# Explain why it's good for the game

We have rare sounds that come from other games like TF2 that have a low chance of triggering. I believe that the metal pipe falling sound fits this category perfectly well. Moreover the sound itself does imitate being thrown at a big metal wall, so it only makes sense.
Yes it has been tested. Yes I got carpal tunnel from that.

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
soundadd: Adds a wall-slam sound that has a very rare chance of triggering.
/:cl:

### CODE REVIEW

**Comment by foxymegneil** [comment]
SPEEDMERGE

File: code/game/sound.dm [hunk 1]
```diff
@@ -213,7 +213,7 @@
 			if("clan_sword_hit")
 				S = pick('sound/weapons/clan_sword_hit_1.ogg', 'sound/weapons/clan_sword_hit_2.ogg')
 			if("slam")
-				S = pick('sound/effects/slam1.ogg','sound/effects/slam2.ogg','sound/effects/slam3.ogg')
+				S = pick('sound/effects/slam1.ogg','sound/effects/slam2.ogg','sound/effects/slam3.ogg', 2;'sound/effects/slam_rare_1.ogg')
```
Comment (harryob) [inline code/game/sound.dm:216]:
```suggestion
				S = pick('sound/effects/slam1.ogg','sound/effects/slam2.ogg','sound/effects/slam3.ogg', 0.1;'sound/effects/slam_rare_1.ogg')
```

**Review by harryob — CHANGES_REQUESTED** [review]
just a bit less. also this isn’t on you but i hate this pick syntax

File: code/game/sound.dm (hunk 1 above)
Comment (Waseemq1235) [reply code/game/sound.dm:216]: oh come on atleast 1

**Comment by Waseemq1235** [comment]
how unstale

**Comment by harryob** [comment]
the funny thing is i think the changed syntax might make it more common. who knows

**Pull request merged**